### PR TITLE
Lightning badger: Limit reported Sentry errors to Glitch.com

### PR DESCRIPTION
**Pull request by oliviacpu**
## Links
* Remix link: https://lightning-badger.glitch.me/
* Manuscript link: https://glitch.manuscript.com/f/cases/3327972/Sentry-exclude-errors-from-browser-extensions
* Specs/Notion docs: n/a

## Changes:
* Adds `glitch.com` to `whitelistUrls` in Sentry initialization to prevent external errors from being reported

## How To Test:
* In`https://glitch.com/edit/#!/lightning-badger?path=shared/sentryHelpers.js:16:0` modify the condition so that errors will be reported when not on production, and then cause an error.

## Feedback I'm looking for:

Configuration.

## Things left to do before deploying:
- [ ] Test

### CODE REVIEW

File: shared/sentryHelpers.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-const onProductionSite = (projectDomain, apiEnvironment) => (projectDomain === 'community' || projectDomain === 'community-staging') && 
-  apiEnvironment === 'production';
+const onProductionSite = (projectDomain, apiEnvironment) =>
+  (projectDomain === 'community' || projectDomain === 'community-staging') && apiEnvironment === 'production';
```
Comment (oliviacpu) [inline shared/sentryHelpers.js]: Prettier.

File: src/components/inputs/text-input.js [hunk 2]
```diff
@@ -12,7 +12,22 @@ const TYPES = ['email', 'password', 'search', 'text'];
 
 const InputPart = ({ children }) => <span className={styles.inputPart}>{children}</span>;
 
-const TextInput = ({ autoFocus, className, disabled, error, labelText, maxLength, name, onChange, opaque, placeholder, postfix, prefix, type, value }) => {
+const TextInput = ({
```
Comment (oliviacpu) [inline src/components/inputs/text-input.js]: Prettier.

File: src/presenters/featured-collections.js [hunk 3]
```diff
@@ -82,6 +81,7 @@ const loadCollection = async (api, { owner, name }) => {
     }
     captureException(error);
   }
+
```
Comment (oliviacpu) [inline src/presenters/featured-collections.js]: Prettier.

File: src/presenters/pop-overs/create-team-pop.js [hunk 4]
```diff
@@ -126,7 +126,13 @@ class CreateTeamPopBase extends React.Component {
 
         <section className="pop-over-actions">
           <form onSubmit={this.handleSubmit}>
-            <TextInput labelText={placeholder} value={this.state.teamName} onChange={this.handleChange} placeholder={placeholder} error={this.state.error} />
+            <TextInput
```
Comment (oliviacpu) [inline src/presenters/pop-overs/create-team-pop.js]: Prettier.

File: src/utils/sentry.js [hunk 5]
```diff
@@ -22,6 +22,7 @@ try {
     environment: ENVIRONMENT,
     release: `community@${BUILD_TIMESTAMP}`,
     ignoreErrors: SentryHelpers.ignoreErrors,
+    whitelistUrls: ['/glitch.com/'],
```
Comment (oliviacpu) [inline src/utils/sentry.js]: The actual change.

**Comment by clottman** [comment]
Will give this a look when I'm done with another PR!

**Pull request merged**